### DISCUSSION TITLE
Let's choose a design system. Vote for all you agree with.

### DISCUSSION THREAD

**Discussion by ehsanmim** [opensheetorg/opensheet#16] [Polls]
Give all the design systems that you like a thumbs up 👍🏻 and those which do not seem good to you a thumbs down 👎🏻.(https://yaytext.com/emoji/thumbs-down/)(https://emojipedia.org/thumbs-down).

**Comment by ehsanmim** [comment]
Coherent Design System (Community)
[Link](https://www.figma.com/community/file/1311381775078816263)
✅Advantages

- It has a huge number of elements and they seem reasonable.

❌Disadvantages
- Doesn't have dark mode

**Comment by ehsanmim** [comment]
Ant Design Open Source (Community)
[Link](https://www.figma.com/community/file/831698976089873405)
✅Advantages
- It has a react library that we can use directly.

❌Disadvantages
- Dark theme costs $1.

**Comment by ehsanmim** [comment]
FREE - Relume Library Figma Kit
[Link](https://www.figma.com/community/file/1078092050664989246)

✅Advantages
- The components are serious and simple.

❌Disadvantages
- The pro version is paid.

**Comment by ehsanmim** [comment]
ADS Components (Community)
[Link](https://www.figma.com/community/file/857042825430121164)

✅Advantages
- It has dark mode.
- It has [React](https://atlassian.design/components/calendar/examples). 
- It is serious. 

❌Disadvantages
- It does not seem so much like social media but a dashboard.

**Comment by ehsanmim** [comment]
Bootstrap 5 UI Kit
[Link](https://www.figma.com/community/file/876022745968684318)
[Link2](https://www.figma.com/community/file/1044316192441037087)

✅Advantages
- It has React.

❌Disadvantages
- Limited number of components.
- A little bit to many sites are written and it is not new.

**Comment by ehsanmim** [comment]
@shadcn/ui - Design System
[Link](https://www.figma.com/community/file/1203061493325953101)

✅Advantages
- It has React.
- It is simple.

❌Disadvantages
- It has limited number of components.
- In Figma we did not find a dark mode.

**Comment by ehsanmim** [comment]
Essential UI - Figma Ui Kit
[Link](https://www.figma.com/community/file/883657911603554993)

✅Advantages
- It is not messy with a lot of pages.
- Easy to locate and copy what you are looking for.

❌Disadvantages
- It is not as vast as some options above.

**Comment by SudoFrost** [comment]
Mecano Design System
[Link](https://www.figma.com/community/file/1172597752450767100)

✅Advantages

- Have dark mode and wireframe
- Components have many variants 
- Components can change by variables

❌Disadvantages

- Beta version
- Not Have a library

**Comment by ehsanmim** [comment]
Dialect – Design System & Component Library (Free)
[Link](https://www.figma.com/community/file/882742539563822709)

✅Advantages
- It is clean and has architecture guide above each component.
- It has tutorials and  well documented 

❌Disadvantages
